.circleci/config.yml

```diff
@@ -215,6 +215,14 @@ jobs:
           # Report but don't fail the build (`|| exit 0`)
           command: npm run typecheck || exit 0
 
+  typecheck-strict:
+    <<: *defaults
+    parallelism: 1
+    steps:
+      - prepare
+      - run:
+          name: TypeScript strict typecheck of individual subprojects
+          command: npm run typecheck -- --project client/landing/gutenboarding
```

Comment:
Keeping to the generic spirit, there might soon be other folders listed here. Once that happens, might make sense to pull them from a file, or keep the list in `package.json` for example by adding `npm run typecheck-strict` or similar?

No action needed for this PR. :-)